parmed/openmm/parameters.py

```diff
@@ -149,7 +149,13 @@ def _remediate_residue_template(cls, params, residue):
         for lonepair in residue.lonepairs:
             lp_atom = lonepair[1]
             params.atom_types[types[lp_atom]].atomic_number = 0
-            atom.atomic_number = 0
+            residue[lp_atom].atomic_number = 0
+
+        # CHARMM Drude force field includes bonds to lone pairs.  Delete them.
+        for bond in list(residue.bonds):
+            if (bond.atom1.atomic_number == 0) or (bond.atom2.atomic_number == 0):
```

Comment:
I think `isinstance(bond.atom1, ExtraPoint)` (imported from `topologyobjects` is a better way of determining whether something is a virtual site or lone pair or something than checking if the atomic number is 0.

Reply:
I tried but it doesn't work.  The atoms are all instances of `Atom` or `DrudeAtom`, not `ExtraPoint`.